storage/store.go

```diff
@@ -238,14 +240,14 @@ var _ multiraft.Storage = &Store{}
 func NewStore(clock *hlc.Clock, eng engine.Engine, db *client.KV, gossip *gossip.Gossip) *Store {
 	s := &Store{
 		StoreFinder: &StoreFinder{gossip: gossip},
-
-		clock:     clock,
-		engine:    eng,
-		db:        db,
-		allocator: &allocator{},
-		gossip:    gossip,
-		closer:    make(chan struct{}),
-		ranges:    map[int64]*Range{},
+		RetryOpts:   defaultRangeRetryOptions,
```

Comment:
It would be nice if we could make zero-valued fields in RetryOptions have the default behavior, so that tests that want to override one option don't have to set all of them (e.g. we're repeating `Constant: 2` in many places now)